p/scripts/extra.js

```diff
@@ -123,6 +123,29 @@ function init_password_observers() {
 	});
 }
 
+// overwrites the href attribute from the url input
+function updateHref() {
+	const urlField = document.getElementById(this.getAttribute('data-input'));
+	const url = urlField.value;
+	if (url.length > 0) {
+		this.href = url;
+		return true;
+	} else {
+		urlField.focus();
```

Comment:
Btw, this line would yield rather unexpected results in combination with mouseover (and even mousedown/keydown). It only goes with click. :-)

Btw, I suspect with the return false here you mean `preventDefault()`?
```suggestion
		e.preventDefault();
		urlField.focus();
```

Reply:
I tried it too, but e.preventDefault() did not work. That is why the `href `will be deleted.

Reply:
That's because this should use `click`.